komodo/release_transpiler.py

```diff
@@ -32,9 +44,26 @@ def build_matrix_file(release_base, release_folder, builtins):
     write_to_file(compiled, f"{release_base}.yml", False)
 
 
-def _build(packages, py_ver, rhel_ver):
+def get_py_coords(release_base: str, release_folder: str) -> list:
+    """Get python versions of release files inside a given release_folder."""
+    filenames_with_prefix = sorted(
+        [
+            filename
+            for filename in os.listdir(release_folder)
+            if filename.startswith(release_base)
+        ]
+    )
+    len_release_base = len(release_base + "-")
+    py_coords = [filename[len_release_base:-4] for filename in filenames_with_prefix]
+    return py_coords
+
+
+def _build(packages: dict, rhel_ver: str, py_ver: str) -> dict:
```

Comment:
can we rename this function? the idea is that we based on rh and py versions pick a specific version for each package, right? what do you think about:

```suggestion
def _package_version_picker(packages: dict, rhel_ver: str, py_ver: str) -> dict:
```

Reply:
> Do you agree with this one?

sure, sounds good!

ohhh i haven't worked with this feature yet from the coder point of view, so i'm not sure, probably you will have to do the rename in other places yourself.